specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/blob.json

```diff
@@ -797,6 +797,14 @@
   "definitions": {
     "ContainerProperties": {
       "properties": {
+        "defaultEncryptionScope": {
```

Comment:
Please add example for this swagger change.

Reply:
Updated